docs/README.md

```diff
@@ -169,12 +169,12 @@ Follow the [quick start Jupyter Notebook](./samples/product_recommendation_demo.
 
 ## 🚀 Roadmap
 
-For a complete roadmap with estimated dates, please [visit this page](https://github.com/linkedin/feathr/milestones?direction=asc&sort=title&state=open).
-
-- [x] Support streaming
-- [x] Support common data sources
+- [x] Support streaming features with transformation
+- [x] Support common data sources and sinks
```

Comment:
The `common` data sources and sinks looks too general to me. Shall we at least add a link to a detail table which concludes all supported / work-in-progress data sources?